packages/app/src/explorer/EntityItem.tsx

```diff
@@ -55,6 +57,37 @@ function EntityItem(props: Props) {
 
           onSelect(path);
         }}
+        onKeyDown={(e) => {
+          switch (e.key) {
+            case 'ArrowDown':
+              focusNext(e);
+              return;
+
+            case 'ArrowUp':
+              focusPrevious(e);
+              return;
```

Comment:
`ArrowDown` and `ArrowUp` handlers were moved here as well so that all handlers are attached at the same place.